openstates/me/bills.py

```diff
@@ -397,10 +403,15 @@ def scrape_vote(self, bill, motion, url):
 
         member_cell = page.xpath("//td[text() = 'Member']")[0]
         for row in member_cell.xpath("../../tr")[1:]:
-            name = row.xpath("string(td[2])")
-            # name = name.split(" of ")[0]
-
+            name = string.capwords(row.xpath("string(td[2])"))
```

Comment:
I think instead of this level of name mangling we'll prefer case insensitive comparisons in the import code